### PR TITLE
config

### PR DESCRIPTION
**Pull request by dickeyxxx**

### CODE REVIEW

**Comment by dickeyxxx** [comment]
can you review this @ransombriggs?

**Comment by ransombriggs** [comment]
@dickeyxxx This code does not escape values the same way the old code did, for example

rbriggs@ebriggs-ltm [08:59:59] [~/git/ruby-getting-started] [master]
-> % heroku config:get k -s
k=v\=1

VS

rbriggs@ebriggs-ltm [09:01:41] [~/git/ruby-getting-started] [master]
-> % heroku config:get k -s       
k=v=1

File: commands/config/set.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+let extend = require('util')._extend;
+let _      = require('lodash');
+
+function* run (context, heroku) {
+  let vars = _.reduce(context.args, function (vars, v) {
+    if (v.indexOf('=') === -1) {
+      cli.error(`${cli.color.cyan(v)} is invalid. Must be in the format ${cli.color.cyan('FOO=bar')}.`);
+      process.exit(1);
+    }
+    v = v.split('=');
```
Comment (ransombriggs) [inline commands/config/set.js]: In v3, you can have a value with an = in it, in v4 it gets truncated.

rbriggs@ebriggs-ltm [09:07:21] [~/git/ruby-getting-started] [master]
-> % heroku config:set k1=foo=bar 
Setting config vars and restarting ancient-atoll-1970... done
rbriggs@ebriggs-ltm [09:07:23] [~/git/ruby-getting-started] [master]
-> % heroku config:get k1
foo
rbriggs@ebriggs-ltm [09:07:31] [~/git/ruby-getting-started] [master]
-> %

Comment (dickeyxxx) [inline commands/config/set.js]: updated

**Pull request merged**